kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/flightsearch/FlightRequestRetriever.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.good.patterns.challenges.flightsearch;
+
+public class FlightRequestRetriever {
```

Comment:
Nie rozumiem, czemu ta klasa się tak nazywa. Konkretnie chodzi o słowo `Request` - kojarzy mi się z requestem od użytkownika. To jest właściwie klasa konfigurująca wyszukiwarkę, a właściwie możesz ją sobie wyobrazić jako "bazę danych" lotów. Poszukaj proszę innej nazwy.